dissect/util/feature.py

```diff
@@ -0,0 +1,54 @@
+import functools
+import os
+from enum import Enum
+from typing import Callable, Optional
+
+
+# Register feature flags in a central place to avoid chaos
+class Feature(Enum):
+    ADVANCED = "advanced"
+    LATEST = "latest"
+    BETA = "beta"
```

Comment:
```suggestion
    LATEST = "latest"
    ADVANCED = "advanced"
    BETA = "beta"
```
Nitpicky, but I believe the "default" flag should be on top.

And maybe experimental over beta? That's mostly personal taste of course, I'm not 100% sure on it either.